spec/api-browser-window-spec.js

```diff
@@ -1106,6 +1106,90 @@ describe('BrowserWindow module', function () {
       })
       w.loadURL('file://' + path.join(fixtures, 'api', 'close-beforeunload-empty-string.html'))
     })
+
+    it('fires for each close attempt', function (done) {
+      var closeCount = 5
+
+      var timeoutHandle = setTimeout(() => {
```

Comment:
Good point, I should have rethought that before I submitted the pull request.  That was all just to workaround issues with the after test hooks not being able to close a page that has a beforeunload handler stopping that.

I'll rework these tests so they're more like the other beforeunload tests (where the beforeunload events stop preventing the window from closing after a fixed number of calls).